Add files via upload

crisforp opened the pull request:
Tasks: creating and managing roles

crisforp committed:
Add files via upload

Tasks: creating and managing roles

mozuela commented:
**Task 1:** More than a generic description of the DB's security, the expected results is the detail of which privilegies are granted to which roles/user and for which tables 
**Task 2-3:** Role name should have underscores instead of spaces, policy create for one user only instead of a role, schema is missed, if not exists/drop if exists is missing, hardcoded user ID, use variable in an anonymous block instead; role creation should be done at role level, no ID/Client level
Task 4: Solid understanding of the concepts, nice work

crisforp committed:
Add files via upload

I made the changes requested by my mentor, with a small addition to task 4:
Task 1: More than a generic description of the DB's security, the expected results is the detail of which privilegies are granted to which roles/user and for which tables
Task 2-3: Role name should have underscores instead of spaces, policy create for one user only instead of a role, schema is missed, if not exists/drop if exists is missing, hardcoded user ID, use variable in an anonymous block instead; role creation should be done at role level, no ID/Client level
Task 4: Solid understanding of the concepts, nice work

mozuela commented:
Hi Cristian, avoid to hardcoded IDs which may not exist, for create a access for a specific client, the expectation is some code it can receive the name of the client and assign/grant the access, in this case it'll only work for Maria Stefania, although an user example must be provider, shouldn't be hardcoded. Please made the udpates and merge

crisforp committed:
Add files via upload

Avoid to hardcoded IDs which may not exist, for create a access for a specific client, the expectation is some code it can receive the name of the client and assign/grant the access, in this case it'll only work for Maria Stefania, although an user example must be provider, shouldn't be hardcoded. Please made the udpates and merge.
